portal/package.json

```diff
@@ -37,8 +37,10 @@
     "@authgear/web": "0.1.0",
     "@fluentui/react": "7.132.0",
     "@oursky/react-messageformat": "0.10.0",
+    "@types/luxon": "^1.24.4",
     "classnames": "2.2.6",
     "history": "5.0.0",
+    "luxon": "^1.25.0",
```

Comment:
Our convention is to use `--save-exact`. Next time you can install new dependency with

```typescript
npm install --save --save-exact [--save-dev] <dep>
```

Now, you can just remove the caret.